libusb/hid.c

```diff
@@ -123,6 +123,10 @@ struct hid_device_ {
 #ifdef DETACH_KERNEL_DRIVER
 	int is_driver_detached;
 #endif
+
+	int error;
+	wchar_t *error_name;
```

Comment:
the way it is used - `error_name` is useless
it would be much more useful to have a single `error_string` formatted as `<libusb_error_name>: <libusb_strerror>`

Reply:
It is simple to compose those two values into a single string later if needed. Parsing them out into two strings is less simple. So I chose to store them separately.

Reply:
That's a fair point. This variable is redundant due to the stored error code. I think I just forgot to delete it.